DentOS_Framework/DentOsTestbed/configuration/testbed_config/basic_vrrp/agg1/DHCP

```diff
@@ -0,0 +1,6 @@
+#
```

Comment:
why create a new testbed for VRR and not enhance one of the existing agg1 and dist1 sit .....

Reply:
We are using agg/infra testbed configs to run functional tests in parallel. Since those tests only require a single device there is no point in expanding the `basic_*` testbed configs.
VRRP tests _can_ be run with the `sit` testbed config, but I am a bit wary of using it since it contains ifupdown configs that might affect the test run. All functional tests expect that the DUT is 'clean' (doesn't contain any bridges/bonds/qdiscs or any ip addresses on swp ports).

Reply:
ifup ifdown should be fine.
i think we should keep as few testbeds definitions as possible to avoid a case where each test requires a dif testbed and then it does not get run as part of SIT